src/models/goal.js

```diff
@@ -22,6 +22,17 @@ module.exports = (sequelize, DataTypes) => {
     status: DataTypes.STRING,
     timeframe: DataTypes.STRING,
     isFromSmartsheetTtaPlan: DataTypes.BOOLEAN,
+    goalNumber: {
+      type: DataTypes.VIRTUAL,
+      get() {
+        const { id, grants } = this;
+        let regionId = 0;
+        if (grants && grants.length > 0) {
+          regionId = grants[0].regionId;
```

Comment:
The `goalNumber` as defined here is kind of goofy because you could have a goal that has grants in multiple regions. However two updates in the future allow us to ignore the goofiness, 1) The goal id format is changing and region is being removed (see comment here https://ocio-jira.acf.hhs.gov/browse/TTAHUB-654) and 2) Goals will be updated to be unique to grants.

Reply:
I think it will be fine here for now. We are currently only using it once they have drilled down to a goal in a certain region.

Down the road this should change as the data changes.

Do you agree?

Reply:
yeah this will change in the future, I don't think we need to worry about it now